src/components/Sidebar/Sidebar.module.css

```diff
@@ -212,8 +215,8 @@ hr {
 
         overflow-y: unset;
```

Comment:
Remove  overflow-y: unset; and then apply filters button will display correctly.